packages/local-environment/tests/nomadenv.int.test.ts

```diff
@@ -0,0 +1,101 @@
+import { expect, assert, use as chaiUse } from "chai";
+import { NomadEnv } from "../src/nomadenv";
+import Docker from "dockerode";
+import { LocalAgent } from "../src/agent";
+import chaiAsPromised from "chai-as-promised";
+import { NomadDomain } from "../src/domain";
+
+chaiUse(chaiAsPromised);
+
+describe("NomadDomain test", () => {
+    //TODO: We should implement any-network connection logic and test accordingly.
+    it('can create a valid NomadEnvironment', async () => {
+        // Creation
+        const le = new NomadEnv({
+            domain: 1,
+            id: "0x" + "20".repeat(20),
+        });
+        expect(le).to.exist;
+        
+        let tDomainNumber = 1;
+        let jDomainNumber = 2;
+      
+        if (process.env.tDomainNumber) {
+          tDomainNumber = parseInt(process.env.tDomainNumber);
+        }
+      
+        if (process.env.jDomainNumber) {
+          jDomainNumber = parseInt(process.env.jDomainNumber);
```

Comment:
I thought we deleted jDomainNumber from .env

Reply:
Oh shoot sorry about that - yes it should be deleted already, removing this.

Reply:
Tysm!